pylint/interfaces.py

```diff
@@ -64,19 +64,15 @@ def implements(
     obj: "BaseChecker",
     interface: Union[Type["Interface"], Tuple[Type["Interface"], ...]],
 ) -> bool:
-    """Return whether the given object (maybe an instance or class) implements
-    the interface.
-    """
+    """Does the given object (maybe an instance or class) implements the interface."""
     kimplements = getattr(obj, "__implements__", ())
     if not isinstance(kimplements, (list, tuple)):
         kimplements = (kimplements,)
     return any(issubclass(i, interface) for i in kimplements)
 
 
 class IChecker(Interface):
-    """This is a base interface, not designed to be used elsewhere than for
-    sub interfaces definition.
-    """
+    """Base interface, not to be used elsewhere than for sub interfaces definition."""
```

Comment:
"not to be used elsewhere other than"

Reply:
Figured we might as well go all in on the typo front : https://github.com/PyCQA/pylint/pull/5575